services/submission/src/main/java/app/coronawarn/server/services/submission/verification/VerificationServerClientConfiguration.java

```diff
@@ -9,9 +8,9 @@
 @Configuration
 public class VerificationServerClientConfiguration {
 
-  private final FeignClientProvider feignClientProvider;
+  private final CloudFeignClientProvider feignClientProvider;
```

Comment:
What's the difference between `CloudFeignClientProvider` and `FeignClientProvider`?

Reply:
`FeignClientProvider` was an abstract interface that was implemented twice, with/without SSL, where `CloudFeignClientProvider` was the SSL version. Since this in now the only remaining version, we use it directly instead of the now obsolete interface.